crons/leaderboards.js

```diff
@@ -1,20 +1,34 @@
 const fs = require('fs/promises');
-const { createPool, sql } = require('slonik');
-const { sleep } = require('./utils/sleep');
-const accountCreationSql = require('./queries/account-creation.sql');
-const scoreSql = require('./queries/score-calculate.sql');
+const { sql } = require('slonik');
+const { sleep } = require('../utils/sleep');
+const accountCreationSql = require('../queries/account-creation.sql');
+const scoreSql = require('../queries/score-calculate.sql');
 
 class LeaderboardCache {
-  constructor(cachePoolConnectionString, indexerPoolConnectionString) {
-    this.cachePool = createPool(cachePoolConnectionString);
+  constructor(statsGalleryCachePool, indexerCachePool, environment) {
+    this.cachePool = statsGalleryCachePool;
```

Comment:
If we're switching to passing in pools instead of connection strings, then this class shouldn't terminate the pools (lines 15-17). I think I would prefer that the cache class manage its own connection pool though. What do you think?